cli/azd/pkg/apphost/generate.go

```diff
@@ -442,7 +450,13 @@ func (b *infraGenerator) LoadManifest(m *Manifest) error {
 			}
 		case "parameter.v0":
 			if err := b.addInputParameter(name, comp); err != nil {
-				return fmt.Errorf("adding input parameter %s: %w", name, err)
+				return fmt.Errorf("adding bicep parameter from resource %s (%s): %w", name, comp.Type, err)
+			}
+		case "value.v0":
+			// getting here means the field value from value.v0 is not empty and has inputs
+			// otherwise it is handled at the beginning of the loop
+			if err := b.addInputParameter(name, comp); err != nil {
+				return fmt.Errorf("adding bicep parameter from resource %s (%s): %w", name, comp.Type, err)
```

Comment:
Can we add some unit tests to validate these additions?

Reply:
Yes, we have a way to do that :).  On it

Reply:
Added generate tests and a sample manifest.

Once preview4 is out, I can add functional tests using @weikanglim test project... which will break after the api changes and will require updates anyway. :)